spec/features/ncr/work_orders/create_spec.rb

```diff
@@ -112,6 +112,17 @@
       expect(page).to_not have_content("Approving official email can't be blank")
     end
 
+    scenario "inactive users do not appear as potential approvers", :js do
```

Comment:
I don't think it hurts to have more tests?

The one I added does use the `:js` feature and clickety stuff to make sure DOM is not hiding it. Not sure if that matters.